### PR TITLE
Fix metabase put logging

### PR DESCRIPTION
**Pull request by alexvanin**
Was broken in #1214.

### CODE REVIEW

File: pkg/local_object_storage/metabase/put.go [hunk 1]
```diff
@@ -462,3 +462,8 @@ func isLinkObject(obj *objectSDK.Object) bool {
 func isLastObject(obj *objectSDK.Object) bool {
 	return len(obj.Children()) == 0 && obj.Parent() != nil
 }
+
+func saddrFromObj(obj *objectSDK.Object) string {
+	return obj.ContainerID().String() + "/" + obj.ID().String()
+
```
Comment (carpawell) [inline pkg/local_object_storage/metabase/put.go:468]: extra line, linter

**Comment by alexvanin** [comment]
> "saddr" means string address?

Yes. Maybe there is already a func to obtain `addressSDK.Address` or `string` from object, but didn't find it.

**Comment by carpawell** [comment]
> didn't find it

maybe it is a sign to add such to SDK? or address getter

**Review by fyrchik — APPROVED** [review]

**Comment by carpawell** [comment]
> Yes. Maybe there is already a func to obtain addressSDK.Address or string from object, but didn't find it.

@alexvanin, https://github.com/nspcc-dev/neofs-node/blob/master/pkg/core/object/object.go#L9

**Pull request merged**